### PR TITLE
ctdump shell add 2 variants

### PR DESCRIPTION
**Pull request by StudioEtrange**
Add two variants wich robustly escape quotes

--shell-form leacy : the historic form

--shell-form form1 : robustly escape quotes

--shell-form form2 : robustly escape quotes AND do not transform list [ a,b ] into separated string

### CODE REVIEW

**Comment by kr41** [comment]
Hey! Thanks for your PRs. 

The project hasn't been released for a long time, so I need to make some preparation before I merge anything. It's mostly about dropping support of dead versions of Python and upgrading developer tools. I'll definitely reformat the code using [Black](https://black.readthedocs.io/en/stable/), so if you don't use it, it's time to start.

Meanwhile, could you provide more context of what this PR does? I don't get which problem does it solve?

**Comment by StudioEtrange** [comment]
Form 1 : same thing that legacy, which is the default behaviour but with escapes quotes for shell
Form 2 : sane thing than form1 BUT I needed to ouput python list without removing [] symbols

**Comment by kr41** [comment]
It doesn't help. 

First of all, `--shell-form form1` and `--shell-form form2` are pretty cryptic and confusing names, which tell nothing about expected result.

About `form1` you wrote it "escapes quotes for shell".  What's wrong with existing formatter? It already escapes quotes for shell. If it escapes them wrong, please create a bug report with appropriate example of wrong behavior.

About `form2` you wrote it "does not transform list [ a,b ] into separated string". It seems, you're trying to push solution of your local problem into main version. I don't see any value it could bring to other users. You can implement your own ad-hoc formatter at `loaderconf.py` file placed at the root of your configs (see the [docs](https://configtree.readthedocs.io/en/latest/advanced_usage.html#loaderconf-py)).

To summarize, the PR mix solutions of two different problems, both of which are unclear. I cannot merge it in its current state.

**Comment by StudioEtrange** [comment]
I will review it later
